<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="toAdd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>AddValue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="toDelete">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>470</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>DeleteValue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_min">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>SearchMinValue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_height">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>SearchHeight</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_max">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="text">
     <string>SearchMaxValue</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_height">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777214</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
